README.md

```diff
@@ -39,10 +39,13 @@ Mac algorithms:
 
 Cipher algorithms:
 * AES/GCM/NoPadding
+* AES_128/GCM/NoPadding
+* AES_256/GCM/NoPadding
 * RSA/ECB/NoPadding
-* RSA/ECB/PKCS1Padding
+* RSA/ECB/Pkcs1Padding
```

Comment:
As per [Standard Algorithm Names](https://docs.oracle.com/javase/9/docs/specs/security/standard-names.html#cipher-algorithm-paddings) we should leave it capitalized as it was before.
```suggestion
* RSA/ECB/PKCS1Padding
```